pkg/agent/agent.go

```diff
@@ -39,10 +95,6 @@ func receiveNewSeccompFile(resolver registry.ResolverFunc, sockfd int) (*registr
 	if err != nil {
 		return nil, nil, fmt.Errorf("cannot parse OCI state: %v\n", err)
 	}
-	seccompFdIndex, ok := containerProcessState.FdIndexes["seccompFd"]
-	if !ok || seccompFdIndex < 0 {
-		return nil, nil, fmt.Errorf("recvfd: didn't receive seccomp fd")
-	}
```

Comment:
Buuut we still leak fds if we exit there: https://github.com/kinvolk/seccompagent/pull/7/files#diff-18bf746206c8ac217eb16ffb0cf91a6e676e05a7b517dc70aef0260d0871028fL40

Reply:
Is this comment still valid? I don't understand where the link should point to.

Reply:
No, I think it is not valid anymore. Sorry I didn't update that :)